packages/@o3r/rules-engine/schemas/rulesets.schema.json

```diff
@@ -166,11 +166,31 @@
             },
             {
               "$ref": "https://raw.githubusercontent.com/AmadeusITGroup/otter/main/packages/%40o3r/components/schemas/rules-engine.placeholder-action.schema.json"
+            },
+            {
+              "$ref": "#/definitions/RulesEngineCustomActionBlock"
             }
           ]
         }
       ]
     },
+    "RulesEngineCustomActionBlock": {
+      "type": "object",
+      "description": "Content of a custom action",
+      "properties": {
+        "actionType": {
+          "type": "string",
+          "description": "Action Type",
+          "pattern": "^(?!(?:SET_FACT|UPDATE_PLACEHOLDER|UPDATE_CONFIG|UPDATE_ASSET|UPDATE_LOCALISATION)$).*$"
```

Comment:
Unfortunately you cannot do it that way without hardcoding value defined in other packages.
There is no way to do a reference instead?